vite.md

```diff
@@ -0,0 +1,370 @@
+
+> {tip} Are you running Laravel Mix? Vite has replaced Laravel Mix in new Laravel installations. For Mix documentation, please visit the [Laravel Mix](https://laravel-mix.com/) website. If you would like to switch to Vite, please see our [upgrade guide](https://github.com/laravel/vite-plugin/blob/main/UPGRADE.md#migrating-from-laravel-mix-to-vite).
+
+<a name="vite-or-mix"></a>
+#### Choosing Between Vite And Laravel Mix
+
+Before transitioning to Vite, new Laravel applications utilized [Mix](https://laravel-mix.com/), which is powered by [webpack](https://webpack.js.org/), when bundling assets. Vite focuses on providing a faster and more productive experience when building rich JavaScript applications. If you are developing a Single Page Application (SPA), including those developed with tools like [Inertia](https://inertiajs.com), Vite will be the perfect fit.
+
+Vite also works well with traditional server-side rendered applications with JavaScript "sprinkles", including those using [Livewire](https://laravel-livewire.com). However, it lacks some features that Laravel Mix supports, such as the ability to copy arbitrary assets into the build that are not referenced directly in your JavaScript application.
+
+<a name="installation"></a>
+## Installation & Setup
+
+> {tip} The following documentation discusses how to manually install and configure the Laravel Vite plugin. However, Laravel's [starter kits](/docs/{{version}}/starter-kits) already include all of this scaffolding! Laravel's starter kits are the fastest way to get started with Laravel and Vite.
+
+<a name="installing-node"></a>
+### Installing Node
+
+You must ensure that Node.js and NPM are installed before running Vite and the Laravel plugin:
+
+```sh
+node -v
+npm -v
+```
+
+You can easily install the latest version of Node and NPM using simple graphical installers from [the official Node website](https://nodejs.org/en/download/). Or, if you are using [Laravel Sail](https://laravel.com/docs/{{version}}/sail), you may invoke Node and NPM through Sail:
+
+```sh
+./vendor/bin/sail node -v
+./vendor/bin/sail npm -v
+```
+
+<a name="installing-vite-and-laravel-plugin"></a>
+### Installing Vite And The Laravel Plugin
+
+Within a fresh installation of Laravel, you will find a `package.json` file in the root of your application's directory structure. The default `package.json` file already includes everything you need to get started using Vite and the Laravel plugin. You may install your application's frontend dependencies by running:
+
+```sh
+npm install
+```
+
+<a name="configuring-vite"></a>
+### Configuring Vite
+
+Vite is configured in a `vite.config.js` file in the root of your project. You are free to customize this file based on your needs and you may also install any other plugins your application requires, such as `@vitejs/vue-plugin` or `@vitejs/react-plugin`.
```

Comment:
The plugins are `@vitejs/plugin-vue` and `@vitejs/plugin-react`

Reply:
Fixed. Thank you!